spacy/lang/ru/lemmatizer.py

```diff
@@ -55,6 +61,7 @@ def pymorphy2_lemmatize(self, token: Token) -> List[str]:
             return [PUNCT_RULES.get(string, string)]
         if univ_pos not in ("ADJ", "DET", "NOUN", "NUM", "PRON", "PROPN", "VERB"):
             # Skip unchangeable pos
+            return self.pymorphy2_lookup_lemmatize(token)
             return [string.lower()]
```

Comment:
Line 65 looks like dead code?

Reply:
Definitely dead code, but I'm not sure what the intention is here with this change vs. comment. The previous behavior was `string.lower()`, but does backing off to the lookup lemmatizer make sense here?

Reply:
Yes, this is the fix from the linked issue 11626. But we should delete line 65.